src/popup/components/AccountCardAdd.vue

```diff
@@ -3,21 +3,23 @@
     class="account-card-add"
     @click="openCreateAccountModal()"
   >
-    <span class="title">
-      <PlusCircle />
-      {{
-        isMultisig
-          ? $t('pages.vaults.addVault')
-          : $t('pages.accounts.addAccount')
-      }}
-    </span>
-    <span class="description">
-      {{
-        isMultisig
-          ? $t('pages.vaults.addVaultDescription')
-          : $t('pages.accounts.addAccountDescription')
-      }}
-    </span>
+    <div class="wrapper">
```

Comment:
Why this wrapper is needed?

Reply:
This fixes the [issue ](https://github.com/aeternity/superhero-wallet/pull/2208#issuecomment-1708128948)reported by Charis. The sliding/clicking on the last card was not working properly.